ApiDoctor.Validation/Tags/TagProcessor.cs

```diff
@@ -180,9 +181,10 @@ public string Preprocess(FileInfo sourceFile)
                                 continue;
                             }
 
-                            var includeContent = Preprocess(includeFile);
-
-                            writer.WriteLine(includeContent);
+                            // Include Files can have Yaml Front Matter as well.
+                            var includeContent = Preprocess(includeFile, issues);
+                            var (_, processedContent) = DocFile.ParseAndRemoveYamlFrontMatter(includeContent, issues);
```

Comment:
All we need to do is remove YAML front matter without parsing it. Parsing it will mean that the included file's front matter should follow all the rules we have for YAML front matter which should not be the case. With this implementation, all documents with included files are throwing errors.